modules/openy_features/openy_demo_content/modules/openy_demo_tcolor/config/install/migrate_plus.migration.openy_demo_taxonomy_term_color.yml

```diff
@@ -14,71 +14,71 @@ source:
     -
       id: 1
       name: White
-      color: 'FFFFFF'
+      color: '#FFFFFF'
     -
       id: 2
       name: Black
-      color: '000000'
+      color: '#000000'
     -
       id: 3
       name: Sky blue
-      color: BFE1F3
+      color: '#BFE1F3'
     -
       id: 4
       name: Light blue
-      color: '00AEEF'
+      color: '#00AEEF'
     -
       id: 5
       name: Blue
-      color: '0089D0'
+      color: '#0089D0'
```

Comment:
How we support upgrade path here?
Will current color vocabularies work?
Should we create hook_update_N for those?

Reply:
I see it works 
![image](https://user-images.githubusercontent.com/563412/93580480-876b4e80-f9a8-11ea-9c1b-efab15898879.png)